prow/istio-pilot-e2e-split-horizon-eds.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export RESOURCE_TYPE="${RESOURCE_TYPE:-gke-e2e-test}"
+export OWNER="istio-multicluster-split-horizon-e2e"
+
+export SETUP_CLUSTERREG="True"
+CLUSTERREG_DIR="${CLUSTERREG_DIR:-$(mktemp -d /tmp/clusterregXXX)}"
+export CLUSTERREG_DIR
+
+./prow/e2e-suite.sh --timeout 50 --cluster_registry_dir="$CLUSTERREG_DIR" --single_test e2e_multicluster_split_horizon "$@"
```

Comment:
This shouldn't be a different prow job from the current multicluster test.   I think it would be best to have as many of the multicluster scenarios tested under the single job as possible.   It's a little tough to do this similar to the way it is done for pilot with many go tests being executed under the same package, because the current multicluster e2e are just pilot tests with an option.   It should be straight forward though to change the current multicluster prow job to execute both tests sequentially in the same environment.

Reply:
Fixed in https://github.com/istio/istio/pull/12709/commits/c2d0ecec903764c93cacf6d6e2780ec2ef01aabe